internal/handlers/pg/pgdb/pool.go

```diff
@@ -343,10 +347,38 @@ func (pgPool *Pool) DropDatabase(ctx context.Context, db string) error {
 	}
 }
 
+// validateCollectionName validates collection name.
+// Collection name must not contain non-latin letters, spaces, dots, dollars, dashes,
+// name collection name length must be less than 119 characters.
+func (pgPool *Pool) validateCollectionName(collection string) error {
```

Comment:
Let's inline this method into the only place where we use it

Reply:
> Let's inline this method into the only place where we use it

removed.